dev/com.ibm.ws.javaee.dd/src/com/ibm/ws/javaee/dd/web/WebApp.java

```diff
@@ -12,7 +12,11 @@
  *******************************************************************************/
 package com.ibm.ws.javaee.dd.web;
```

Comment:
Of course this is an SPI package so adding things to this class is going to trigger a version update I think.  I guess not having it here may be better?  So moving the new field method from this class to a static method in WebFragmentsInfoImpl may be best.

Reply:
I think I'll need to do that anyway since I forgot to update WebFragment to include 6.1.
Is there a downside to updating the version?

Reply:
The bigger question is do we want to expose this new method and Set to the SPI interface?

Reply:
Ahh I see.

Reply:
WebFragment didn't add any new methods or fields, so I wouldn't expect a version update.